docs/docs/middleware.md

```diff
@@ -1,14 +1,13 @@
 # Middleware
 
-A middleware is a wrapper around a [service] that provides a means of manipulating
```

Comment:
I can be biased, but explaining things using 'wrapper' things doesn't help much.